agenta-cli/agenta/sdk/decorators/llm_entrypoint.py

```diff
@@ -55,20 +55,18 @@
 class entrypoint(BaseDecorator):
     """Decorator class to wrap a function for HTTP POST, terminal exposure and enable tracing.
 
-    Args:
-        BaseDecorator (object): base decorator class
 
     Example:
     ```python
         import agenta as ag
 
-        @ag.entrypoint(enable_tracing=True) # Defaults to False
+        @ag.entrypoint
         async def chain_of_prompts_llm(prompt: str):
             return ...
     ```
     """
 
-    def __call__(self, func: Callable[..., Any]):
+    def __init__(self, func: Callable[..., Any]):
```

Comment:
Is there any reason you changed this to `__init__`?

Reply:
Yes, so that we don't have to call `ag.entrypoint()` but call `ag.entrypoint`